RELEASING.md

```diff
@@ -0,0 +1,21 @@
+# Release process
+
+Testcontainers' release process is automated as a Travis deployment. This describes the basic steps for a project member to perform a release.
+
+## Steps
+
+1. Ensure that the master branch is building and that tests are passing.
+1. Check that the [`CHANGELOG`](CHANGELOG.md) file is up to date and includes all merged features.
+1. Create a tag for the new release with the desired version number. **The tag name is used as the Maven version**, so please keep the tag name plain (e.g. 1.2.3).
```

Comment:
I would change 3 & 4 to "Create a new release on GitHub" (it will create the tag also, less actions, no need to clone & push locally, etc)